src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((stylesObject, style) => {
+      const [property, value] = style.split(':');
```

Comment:
Instead of unpacking the object on each iteration, we simply add a new property to the existing object.
```suggestion
      const [property, value] = style.split(':').map(part => part.trim());
```